src/building/bootstrapping/what-bootstrapping-does.md

```diff
@@ -372,6 +372,7 @@ You can find more discussion about sysroots in:
   Zulip](https://rust-lang.zulipchat.com/#narrow/stream/182449-t-compiler.2Fhelp/topic/deps.20in.20sysroot/)
 - [Discussions about building rustdoc out of
   tree](https://rust-lang.zulipchat.com/#narrow/stream/182449-t-compiler.2Fhelp/topic/How.20to.20create.20an.20executable.20accessing.20.60rustc_private.60.3F)
+- [This issue](https://github.com/rust-lang/rust/issues/137421) explaining that `rustc_private` linker fails sometimes because llvm-tools is not installed.
```

Comment:
This is not the right place to put this remark. Notably, this is the build system chapter but `rustc_private` is a perma-unstable compiler feature. This should go somewhere near https://rustc-dev-guide.rust-lang.org/rustc-driver/intro.html. Maybe just add a new "Remarks on perma-unstable features" chapter next to this chapter?